### PR TITLE
Login button

### PR DESCRIPTION
**Pull request by XLuna24**
Add Login/Register button to navigation bar, which also changes to logout when user is authenticated. Users username appears  in the navigation bar when authenticated.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -7,9 +7,12 @@
     <script src="home.js"></script>
   </head>
 
-  <body>
+  <body onload="loadLogin()">
       <div class="topnav">
           <h1>Mediaphile</h1>
+          <a id="userName"></a>
+          <a class="button" id="user-login">Login/Register</a>
```
Comment (henryksloan) [inline src/main/webapp/index.html:14]: I still think "Sign in with Google" would be best, especially if you use the official "Sign in with Google" image. At least until we possibly change the login journey so it matches the distinction between register and login.

File: src/main/webapp/home.css [hunk 2]
```diff
@@ -15,6 +15,19 @@ html, body {
     float:left;
 }
 
+#userName {
```
Comment (henryksloan) [inline src/main/webapp/home.css:18]: I think this should be`username` with no caps to match what the api returns.

File: src/main/webapp/index.html (hunk 1 above)
Comment (XLuna24) [reply src/main/webapp/index.html]: Updated button to "Sign in with Google" button

File: src/main/webapp/home.css (hunk 2 above)
Comment (XLuna24) [reply src/main/webapp/home.css]: fixed

**Pull request merged**